### ISSUE TITLE
Missing label for external link URL field

### ISSUE THREAD

**Issue by acarasimon96** [wagtail/wagtail#5656]
### Issue summary

When adding an external link through the Draftail rich text editor, the label for the URL field only shows a red asterisk. Before Wagtail 2.7, the text "URL:" appears before that asterisk. This is only seen in the "External Link" tab of the page chooser modal but not in other similar tabs such as "Email link."

![image](https://user-images.githubusercontent.com/17580077/67608821-2b238e80-f747-11e9-807c-e7db8870034c.png)

### Steps to reproduce

1. Edit any page with rich text from the admin
2. Click on the link icon in a Draftail editor
3. Click on "External link"
4. Note the missing label on the first textfield

### Further details

* The label is not hidden -- in fact, it is missing in the page's HTML as seen from Chrome's DevTools inspector:
  ```html
  <div class="field url_or_absolute_path_field text_input">
    <label for="id_external-link-chooser-url"></label>
    <!-- ... -->
  </div>
  ```
* In addition, when I enter text in the above element with DevTools and switch between to a different tab, that text disappears.
* There were no error messages in the DevTools console while reproducing the bug.
* I can see the label just fine in a production version of my project running Wagtail 2.6 (or a new project using that version), so with some research, I believe the bug was introduced in #5478.
* Since I installed/upgraded to a prerelease version of Wagtail via `pip install`, I compiled admin assets per the [docs](https://docs.wagtail.io/en/stable/contributing/developing.html#setting-up-the-wagtail-codebase).
* I may submit a pull request that resolves this issue myself, but it won't be submitted immediately because I have to figure out what caused it (I sadly don't have any experience with React).
* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: yes (I made no modifications to the Draftail frontend code in my project)

### Technical details

* Python version: 3.6.7
* Django version: 2.2.4
* Wagtail version: 2.7rc2
* Browser version: Chrome 78

**Comment by fefi95** [comment]
Hi! I would like to work on this

**Comment by lb-** [comment]
Resolved & merged in - 34db9a394fb6a0b14b26b36f04310a3eb5b50867 & 92864f22b4d878b0c40bd5f5ef9a74acd0c542b3

via #5659
